### ISSUE TITLE
Unable to launch kernel

### ISSUE THREAD

**Issue by bajazid** [aionnetwork/aion#7]
Hi,
I got following error after I run aion.sh:

    # A fatal error has been detected by the Java Runtime Environment:
    #
    #  SIGILL (0x4) at pc=0x00007f8188ae7443, pid=15965, tid=16035
    #
    # JRE version: Java(TM) SE Runtime Environment (9.0+11) (build 9.0.4+11)
    # Java VM: Java HotSpot(TM) 64-Bit Server VM (9.0.4+11, mixed mode, tiered, compressed oops, g1 gc, linux-amd64)
    # Problematic frame:
    # C  [equiMiner.so+0x2443]  equi::listindices1(unsigned int, tree, unsigned int*)+0x33
    #
    # Core dump will be written. Default location: Core dumps may be processed with "/usr/share/apport/apport %p %s %c %d %P" (or dumping to /home/xxxxxxxxxx)
    #
    # An error report file with more information is saved as:
    # /home/xxxxxxxxxxx/hs_err_pid15965.log
    [thread 16034 also had an error]
    #
    # If you would like to submit a bug report, please visit:
    #   http://bugreport.java.com/bugreport/crash.jsp
    #
    ./aion.sh: line 59: 15965 Aborted                 (core dumped) env EVMJIT="-cache=1" ./jre/bin/java -Xms2g --add-modules=java.xml.bind -cp "./lib/*" org.aion.Aion "$@"

Version 0.1.8
Host: Intel(R) Core(TM) i7-3687U CPU @ 2.10GHz, 4 cores, 15G, Ubuntu 17.10

Any idea what could be wrong here?

**Comment by aion-jin** [comment]
@aion-Ross    could you check if equiMiner.so build parameter support cpu without avx support ?

**Comment by hbarclay** [comment]
Hello, I get this same error after the following steps:

- Configure and run aion.sh
- Wait for sync
- During sync (in fact after sync completes), program throws nullptr exception and freezes
- Kill process, restart aion.sh
- The above error is shown


The exception that was thrown:
```
Exception in thread "process-in" java.lang.NullPointerException
    at org.aion.p2p.a0.P2pMgr.processInboundTcpNetRoute(P2pMgr.java:372)
    at org.aion.p2p.a0.P2pMgr.processInboundTcp(P2pMgr.java:481)
    at org.aion.p2p.a0.P2pMgr.lambda$runProcessInbound$3(P2pMgr.java:563)
    at java.base/java.lang.Thread.run(Unknown Source)
```

Running in VirtualBox on Ubuntu 16.04

**Comment by MrYukonC** [comment]
I am getting the same error as @bajazid in the original post.

However, if I disable mining in config.xml file the error does not happen.

Also, I cannot tell if I am successfully syncing or not.

I have all of the show-status flags set to true config.xml, but the only output I see is the following:

> ****************p2p--1108564543********************
<net inbound-nodes(channelHash)=[]>
<net outbound-nodes(nodeIdHash)=[]>
<net active-nodes(nodeIdHash)=[
> 
> ]>
****************p2p--1108564543********************
<net inbound-nodes(channelHash)=[]>
<net outbound-nodes(nodeIdHash)=[]>
<net active-nodes(nodeIdHash)=[
> 
> ]>
****************p2p--1108564543********************
<net inbound-nodes(channelHash)=[]>
<net outbound-nodes(nodeIdHash)=[]>
<net active-nodes(nodeIdHash)=[
> 
> ]>
****************p2p--1108564543********************
<net inbound-nodes(channelHash)=[]>
<net outbound-nodes(nodeIdHash)=[]>
<net active-nodes(nodeIdHash)=[
> 
> ]>
> ...

**Comment by bajazid** [comment]
And here is error report file
[hs_err_pid15965.log](https://github.com/aionnetwork/aion/files/1714764/hs_err_pid15965.log)

**Comment by gonuco** [comment]
thanks @hbarclay  the bug you reported already fixed in weekly release,  you will get an update email with new binaries.

thanks @MrYukonC @bajazid ,  the mining thread crash jvm bug , team will working on it.

**Comment by bajazid** [comment]
@gonuco this seems to be fixed in the latest version (0.1.10). I managed to sync all blocks and started mining without any issues. I've tried "clean" shutdown and killing the process, but error from above never showed up.
